### PR TITLE
Update operating procedures

### PR DESCRIPTION
**Pull request by jpradocueva**
This PR provides an update to the operating procedures based on the discussion and suggestions during the meeting.

### CODE REVIEW

File: operating_procedures.md
Comment (shawnalpay) [inline operating_procedures.md:87]:
```suggestion 
The FG Maintainer(s) MUST apply the "Review & Approval" process — covered in this document — to contributions submitted by the Working Group members.
```

File: operating_procedures.md
Comment (shawnalpay) [inline operating_procedures.md:96]: Each Github Issue in the scope of the Release Version under development SHOULD be sponsored by at least one FG Maintainer.  At least one of the Maintainer sponsors of an Issue MUST:

- Assemble a team to consider and development the Issue
- Provide technical guidance and oversight for development
- Consistently attend the recurring Task Force meetings to which the Issue has been assigned
- Provide progress updates in recurring Task Force meetings
- Help unblock technical hurdles encountered during development
- Ensure the timely resolution of the Issue, based on the release planning timeline

File: operating_procedures.md
Comment (jpradocueva) [inline operating_procedures.md:53]: Change  [RELEASE_PLANNING] to [RELEASE-PLANNING]

File: operating_procedures.md
Comment (jpradocueva) [inline operating_procedures.md:89]: Add a link to `/workflow-procedures.md`

**Review by jpradocueva — COMMENTED** [review]
Added comments that will require further changes to the PR.

File: operating_procedures.md
Comment (jpradocueva) [inline operating_procedures.md:53]:
```suggestion
A FOCUS Contributors MUST read the Project documentation (e.g.: this operational document, contribution guidelines, [README](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/README.md), [CONTRIBUTING](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/CONTRIBUTING.md), and [RELEASE-PLANNING](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/RELEASE-PLANNING.md) documents) before attempting to submit an issue or pull request.
```

File: operating_procedures.md
Comment (jpradocueva) [inline operating_procedures.md:85]:
```suggestion
The FG Maintainer(s) MUST keep the project documentation up to date (e.g.: CONTRIBUTING, README and RELEASE PLANNING documents), and MUST have a good working knowledge of GitHub and other Project tools in use.
```

**Review by jpradocueva — COMMENTED** [review]
Comments incorporated.

**Comment by jpradocueva** [comment]
### Action Items from the Maintainers' meeting on December 9:
- [ ] [PR [#17](https://github.com/FinOps-Open-Cost-and-Usage-Spec/foundation/pull/17)] All members to review the proposed changes and provide comments or suggestions.

File: operating_procedures.md
Comment (cnharris10) [inline operating_procedures.md:53]:
```suggestion
A FOCUS Contributor MUST read the Project documentation (e.g.: this operational document, contribution guidelines, [README](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/README.md), [CONTRIBUTING](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/CONTRIBUTING.md), and [RELEASE-PLANNING](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/RELEASE-PLANNING.md) documents) before attempting to submit an issue or pull request.
```

File: operating_procedures.md
Comment (cnharris10) [inline operating_procedures.md:72]:
```suggestion
  FOCUS Group (FG) Maintainers are a subset of the FG members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
```

File: operating_procedures.md
Comment (cnharris10) [inline operating_procedures.md:80]: I think this should be 2 or more cycles

File: operating_procedures.md
Comment (cnharris10) [inline operating_procedures.md:81]: Would look for some guiding examples here.

Ex: Some examples may include leading a large task force topic or project to completion, expanding or modifying spec columns, attributes, or guidance in a non-nominal way, etc.

Comment (tobrien) [reply operating_procedures.md:81]: It might make sense to leave some leeway for defining what "consistently" is @cnharris10 - one challenge of defining minimum participation is that it may end up encouraging people to meet just that bar.

**Review by tobrien — APPROVED** [review]
There are some language simplification changes that we can put in another PR, but I support this PR as-is.

File: operating_procedures.md
Comment (mike-finopsorg) [inline operating_procedures.md:83]: i think it would be helpful to be clearer on when a maintainer will be "at risk" of being removed. This line is a little vague, it think something like "at the end of each release cycle, a review of maintainers contributions will be held and active maintainers will maintain their status going into the next release cycle" (obviously with better words). Maybe also something about being able to pause your status and resume if prior notice is provided (basically we understand life gets in the way, just don't drop off the face of the planet let us know you need to pause).....

File: operating_procedures.md
Comment (shawnalpay) [reply operating_procedures.md:80]: Disagree.  If someone has a strong showing i/t/o spec contribution over the course of a single release, we should consider designating them; not automatically of course, but they would be eligible.  If two or more cycles were the rule, then I would not have been eligible to be designated a Maintainer after 1.0 (though maybe you would have preferred that!!).

File: operating_procedures.md
Comment (shawnalpay) [reply operating_procedures.md:81]: Agree with @tobrien; the idea here was to give leeway to the WG chair to use good judgement in making the determination for designation.

File: operating_procedures.md
Comment (shawnalpay) [reply operating_procedures.md:83]: I think your concern is sufficiently handled in line 76, which includes:

> Maintainer designations will be revisited at the discretion of the FG chair, typically at the conclusion of a release cycle. 

I think that "discretion" handles for such cases recurring contribution and personal leave.  Let me know if you disagree.

**Comment by jpradocueva** [comment]
The PR is agreed during the Members' call on December 19.

**Pull request merged**
